### PR TITLE
update plugin.json to adapter next release

### PR DESCRIPTION
**Pull request by peterhashair**
### Description:

Issue https://github.com/matomo-org/matomo/issues/19768. 

Release version 4.3.0 of the plugin which needs min Matomo 4.12

update plugin.json to adapter next release

### Review

* [ ] [Functional review done](https://developer.matomo.org/guides/pull-request-reviews#functional-review-done)
* [ ] [Potential edge cases thought about](https://developer.matomo.org/guides/pull-request-reviews#potential-edge-cases-thought-about) (behavior of the code with strange input, with strange internal state or possible interactions with other Matomo subsystems)
* [ ] [Usability review done](https://developer.matomo.org/guides/pull-request-reviews#usability-review-done) (is anything maybe unclear or think about anything that would cause people to reach out to support)
* [ ] [Security review done](https://developer.matomo.org/guides/security-in-piwik#checklist)
* [ ] [Wording review done](https://developer.matomo.org/guides/pull-request-reviews#translations-wording-review-done)
* [ ] [Code review done](https://developer.matomo.org/guides/pull-request-reviews#code-review-done)
* [ ] [Tests were added if useful/possible](https://developer.matomo.org/guides/pull-request-reviews#tests-were-added-if-usefulpossible)
* [ ] [Reviewed for breaking changes](https://developer.matomo.org/guides/pull-request-reviews#reviewed-for-breaking-changes)
* [ ] [Developer changelog updated if needed](https://developer.matomo.org/guides/pull-request-reviews#developer-changelog-updated-if-needed)
* [ ] [Documentation added if needed](https://developer.matomo.org/guides/pull-request-reviews#documentation-added-if-needed)
* [ ] Existing documentation updated if needed

### CODE REVIEW

File: plugin.json [hunk 1]
```diff
@@ -1,10 +1,10 @@
 {
   "name": "DeviceDetectorCache",
   "description": "Makes tracking faster by detecting many devices, operating systems, bots, and browsers from a cache.",
-  "version": "4.2.3",
+  "version": "4.2.4",
```
Comment (peterhashair) [inline plugin.json]:
```suggestion
  "version": "4.2.5",
```

File: plugin.json [hunk 2]
```diff
@@ -1,10 +1,10 @@
 {
   "name": "DeviceDetectorCache",
   "description": "Makes tracking faster by detecting many devices, operating systems, bots, and browsers from a cache.",
-  "version": "4.2.3",
+  "version": "4.2.5",
```
Comment (peterhashair) [inline plugin.json]:
```suggestion
  "version": "4.3.0",
```

**Comment by AltamashShaikh** [comment]
@peterhashair can we update `https://github.com/matomo-org/plugin-DeviceDetectorCache/blob/4.x-dev/CHANGELOG.md` also with `4.3.0` change ?

**Review by AltamashShaikh — APPROVED** [review]

File: plugin.json [hunk 3]
```diff
@@ -1,10 +1,10 @@
 {
   "name": "DeviceDetectorCache",
   "description": "Makes tracking faster by detecting many devices, operating systems, bots, and browsers from a cache.",
-  "version": "4.2.3",
+  "version": "4.3.0",
   "theme": false,
   "require": {
-    "matomo": ">=4.0.0-rc1,<5.0.0-b1"
+    "matomo": ">=4.12.0-rc1,<5.0.0-b1"
```
Comment (peterhashair) [inline plugin.json]:
```suggestion
    "matomo": ">=4.12.0-b3,<5.0.0-b1"
```

**Pull request merged**